Update app registration warning in Azure setup (#32803)
DataDog/documentation: content/en/getting_started/integrations/azure.md

5. Start the deployment process on the [Review + create tab][33].

> <div class="alert alert-danger"><a href="https://docs.datadoghq.com/logs/log_configuration/archives/?tab=azurestorage">Log Archives</a> require the app registration setup method. For US3 Datadog accounts using the Azure Native integration, follow this page’s steps to [create an app registration](#setup).

Reviewer: ```suggestion
<div class="alert alert-danger"><a href="https://docs.datadoghq.com/logs/log_configuration/archives/?tab=azurestorage">Log Archives</a> require the app registration setup method. For US3 Datadog accounts using the Azure Native integration, see [Create an app registration](#setup).
```
